### PR TITLE
CLI Readme

### PR DESCRIPTION
**Pull request by shaikatz**

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Kamus CLI
+
+This cli was created to give an easy interface to interact with Kamus API.
+
+It supports azure device flow authentication out of the box.
```
Comment (omerlh) [inline cli/README.md]: Link?

File: cli/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+#### Supported commands:
+
+##### Encrypt
+`node index.js encrypt <data> <serviceAccount> <namespace>`
+
+---
+#### How to enable azure active directory authentication 
+You need working active directory tenant and designated app registration, Then just set all the `auth` prefixed options.
```
Comment (omerlh) [inline cli/README.md]: Add link to how to create such an app?

File: cli/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Once the user will run the cli with the auth options, he will get a small code and and azure URL to login into.
+
+---
+##### CLI options:
```
Comment (omerlh) [inline cli/README.md]: Write a few words on each one of the commands?

**Comment by shaikatz** [comment]
@omerlh Fixed the issues, review again?

**Pull request merged**